### PR TITLE
Added Flow.startFeature and Flow#startFeature

### PR DESCRIPTION
**Pull request by smith-kyle**

### CODE REVIEW

File: spec/utils/RepoUtils.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  repoDir: '../repos',
+
+  addFileToIndex(repository, fileName) {
+  	return repository.openIndex()
+  		.then((index) => {
+  			index.read(1);
+  			index.addByPath(fileName);
+  			index.write();
+  			return index.writeTree();
+  		});
```
Comment (jdgarcia) [inline spec/utils/RepoUtils.js]: did these indentations actually pass the linter?

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "babel-cli": "^6.3.15",
     "babel-preset-es2015": "^6.3.13",
     "eslint": "^1.9.0",
+    "fs-extra": "^0.26.2",
```
Comment (jdgarcia) [inline package.json]: What function do we need that we can't just use `fs`?

Comment (smith-kyle) [inline package.json]: `ensureDirSync`

File: spec/utils/RepoUtils.js (hunk 1 above)
Comment (smith-kyle) [inline spec/utils/RepoUtils.js]: lol yeah

Comment (smith-kyle) [inline spec/utils/RepoUtils.js]: well it's fixed now anyways. Are you looking to add another lint rule?

File: package.json (hunk 2 above)
Comment (jdgarcia) [inline package.json]: :+1:

File: spec/utils/RepoUtils.js (hunk 1 above)
Comment (jdgarcia) [inline spec/utils/RepoUtils.js]: I can add it later, outside this PR.

**Pull request merged**